website/docs/guides/setting-up-dex.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Configuring OIDC with Dex and GitHub
+sidebar_position: 3
+---
+
+In this guide we will show you how to enable users to login to the Weave GitOps dashboard by authenticating with their GitHub account.
+
+This example uses [Dex][tool-dex] and its GitHub connector, and assumes Weave GitOps has already been installed on a Kubernetes clusters.
+
+### Pre-requisites
+- A Kubernetes cluster such as [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) cluster running a 
+[Flux-supported version of Kubernetes](https://fluxcd.io/docs/installation/#prerequisites)
+- Weave GitOps is [installed](../installation.mdx) and [TLS has been enabled](../configuration/tls.md).
+
+## What is Dex?
+
+[Dex][tool-dex] is an identity service that uses [OpenID Connect][oidc] to
+drive authentication for other apps.  
+
+Alternative solutions for identity and access management exist such as [Keycloak](https://www.keycloak.org/).
+
+[tool-dex]: https://dexidp.io/
+[oidc]: https://openid.net/connect/
+
+## Create Dex namespace
+
+Create a namespace where Dex will be installed:
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dex
+```
+
+## Add credentials
+
+There are a [lot of options][dex-connectors] available with Dex, in this guide we will
+use the [GitHub connector][dex-github].
+
+We can get a GitHub ClientID and Client secret by creating a
+[new OAuth appliation][github-oauth].
+
+TODO - should we provide more detail for the above?
```

Comment:
TODO? which above?

Reply:
/summon @sympatheticmoose

Reply:
i guess i'll just attach an image about the oauth app